Services/GEV/Reports/classes/class.catBasicReportGUI.php

```diff
@@ -244,64 +244,55 @@ protected function makeGroupKey($row) {
 		return $head.$tail;
 	}
 
-	protected function exportXLS() {
-		require_once "Services/Excel/classes/class.ilExcelUtils.php";
-		require_once "Services/Excel/classes/class.ilExcelWriterAdapter.php";
-		
-		$data = $this->getData();
+	/**
+	* provide xlsx version of report for download.
+	*/
+	protected function exportXLSX() {
+		require_once 'Services/ReportsRepository/classes/class.catXLSXWriter.php';
 
-		$adapter = new ilExcelWriterAdapter("Report.xls", true); 
-		$workbook = $adapter->getWorkbook();
-		$worksheet = $workbook->addWorksheet();
-		$worksheet->setLandscape();
+		$workbook = new catXLSXWriter();
```

Comment:
Nope, bitte das new catXLSSWriter in eine Methode catBasicReportGUI::getExcelWriter auslagern. Dann könnten wir die Methode fast mit einem Mock testen.

Wir haben außerdem zusammen festgestellt, dass der Name der Methode dann immer noch nicht richtig ist und die Methode eher exportExcel heißen müsste.